frontend/packages/kubevirt-plugin/src/components/vms/vm-resource.tsx

```diff
@@ -145,7 +145,7 @@ export const VMDetailsList: React.FC<VMResourceListProps> = ({
         idValue={prefixedID(id, 'vm-statuses')}
       >
         <VMStatusModal isOpen={isStatusModalOpen} setOpen={setStatusModalOpen} vmi={vmi} />
-        <VMStatuses vm={vm} vmi={vmi} pods={pods} migrations={migrations} />
+        <VMStatuses vm={isVM && vm} vmi={vmi} pods={pods} migrations={migrations} />
```

Comment:
we shouldn't have a different approach for vm row and the details page.

Also we should try to infer the status from all the connected resources we are aware of. There might be some bug or wrong implementation in getVMStatus.  We should be consistent for all statuses.

Reply:
done